rapyuta_io_sdk_v2/client.py

```diff
@@ -187,3 +187,1012 @@ def get_project(self, project_guid: str = None) -> Munch:
+        if project_guid:
+            headers["project"] = project_guid
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.delete(
+            url=f"{v2api_host}/v2/disks/{name}/",
+            headers=headers,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
+
+    def list_staticroutes(self, project_guid: str = None) -> Munch:
+        """List all static routes in a project.
+
+        Returns:
+            Munch: List of static routes as a Munch object.
+        """
+        headers = self.config.get_headers()
+        if project_guid:
+            headers["project"] = project_guid
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.get(
+            url=f"{v2api_host}/v2/staticroutes/",
+            headers=headers,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
+
+    def create_staticroute(self, project_guid: str, body: dict) -> Munch:
+        """Create a new static route.
+
+        Returns:
+            Munch: Static route details as a Munch object.
+        """
+        headers = self.config.get_headers()
+        if project_guid:
+            headers["project"] = project_guid
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.post(
+            url=f"{v2api_host}/v2/staticroutes/",
+            headers=headers,
+            json=body,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
+
+    def get_staticroute(self, project_guid: str, name: str) -> Munch:
+        """Get a static route by its name.
+
+        Args:
+            name (str): Static route name
+
+        Returns:
+            Munch: Static route details as a Munch object.
+        """
+        headers = self.config.get_headers()
+        if project_guid:
+            headers["project"] = project_guid
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.get(
+            url=f"{v2api_host}/v2/staticroutes/{name}/",
+            headers=headers,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
+
+    def update_staticroute(self, project_guid: str, name: str, body: dict) -> Munch:
+        """Update a static route by its name.
+
+        Args:
+            name (str): Static route name
+            body (dict): Update details
+
+        Returns:
+            Munch: Static route details as a Munch object.
+        """
+        headers = self.config.get_headers()
+        if project_guid:
+            headers["project"] = project_guid
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.put(
+            url=f"{v2api_host}/v2/staticroutes/{name}/",
+            headers=headers,
+            json=body,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
```

Comment:
thought(non-blocking): I'm thinking we can move the handle_server_errors and munchify part into a decorator. So the methods will be much more straightforward.

```
headers = self.config.get_headers(**kwargs)
return self.c.put(url=f"{self.api_host}/v2/staticroutes/{name}/", headers=headers, json=body)
```